### PR TITLE
Include ChainerCV doc

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

**Comment by takagi** [comment]
Would you check the CI failure?

**Comment by kmaehashi** [comment]
Makefiles are not used when publishing to RTD. Another hack may be needed.
https://readthedocs.org/projects/chainer/builds/7918379/

**Comment by yuyu2172** [comment]
jenkins, test this please.

**Comment by yuyu2172** [comment]
jenkins, test this please.

File: docs/source/conf.py [hunk 1]
```diff
@@ -365,6 +368,23 @@ def setup(app):
     app.connect('autodoc-process-docstring', _autodoc_process_docstring)
     app.connect('build-finished', _build_finished)
 
+    chainercv_version = '0.10.0'
```
Comment (takagi) [inline docs/source/conf.py]: How about drawing chainercv version from the installed one? (also document sources?)

Comment (yuyu2172) [reply docs/source/conf.py]: OK. I will change the code to use the version of ChainerCV that is installed.
Is it OK to hardcode version in `setup.py` for `docs` install mode?

**Comment by yuyu2172** [comment]
Can someone review https://github.com/chainer/chainer-test/pull/457?

File: docs/source/conf.py (hunk 1 above)
Comment (takagi) [reply docs/source/conf.py]: I think it is Okay with leaving a comment as `# see chainer/5434` .

**Comment by yuyu2172** [comment]
jenkins, test this please.

**Comment by beam2d** [comment]
Jenkins, test this please.

**Comment by beam2d** [comment]
Jenkins, test this please.

**Comment by takagi** [comment]
Currently no doctest runs on chainercv repo?

**Comment by yuyu2172** [comment]
No..

With the current implementation, Chainer would need to be able to import the latest released version of ChainerCV. This assumption can fail when functions imported by ChainerCV get deleted in Chainer's master branch.
I talked with beam2d-san offilne, and we decided not to integrate docs. Instead I will add links to ChainerCV's docs hosted here http://chainercv.readthedocs.io/.

File: docs/source/reference/links.rst [hunk 2]
```diff
@@ -126,6 +126,10 @@ VGG Networks
    chainer.links.VGG19Layers
    chainer.links.model.vision.vgg.prepare
 
+.. note::
+   ChainerCV contains implementation of VGG networks as well (i.e.,
+   :class:`chainercv.links.model.vgg.VGG16`).
```
Comment (kmaehashi) [inline docs/source/reference/links.rst]: Could you clarify the difference from Chainer's one? Users may want to know which one is what they want.

Comment (yuyu2172) [reply docs/source/reference/links.rst]: Thank you for the comment.

**Comment by yuyu2172** [comment]
Jenkins, test this please.

**Comment by takagi** [comment]
LGTM!

**Pull request merged**